egui_demo_lib/src/apps/demo/table_demo.rs

```diff
@@ -75,7 +75,7 @@ impl TableDemo {
                     });
                 });
             })
-            .body(|mut body| {
+            .body(|body| {
                 if self.virtual_scroll {
```

Comment:
note: technically both branches here use virtual scroll, so it might be helpful to come up with a different distinction here. while doing that it could also be helpful from a development standpoint to make the number of rows (in either case) configurable to make it easier to test edge cases

Reply:
I think it would be good to demo all three version - the simple version for short tables, the version with homogeneous row heights, and the new heterogeneous version